code/game/machinery/doors/airlock.dm

```diff
@@ -851,11 +851,16 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/proc/toggle_bolt(mob/user)
 	if(wires.is_cut(WIRE_DOOR_BOLTS))
 		to_chat(user, "<span class='warning'>The door bolt control wire has been cut - Door bolts permanently dropped.</span>")
-	else if(lock())
-		to_chat(user, "<span class='notice'>The door bolts have been dropped.</span>")
-	else if(unlock())
+		return
```

Comment:
If youre turning one into an early return, either do all of them or none of them

Reply:
Left it that way since `if(unlock) else if(lock)` made sense to me, but may as well change it for consistency, sure.